Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Kimai - Time tracking/Main application](https://hosted.weblate.org/projects/kimai/messages/).


It also includes following components:

* [Kimai - Time tracking/Plugins](https://hosted.weblate.org/projects/kimai/plugins/)

* [Kimai - Time tracking/Teams](https://hosted.weblate.org/projects/kimai/teams/)

* [Kimai - Time tracking/Emails](https://hosted.weblate.org/projects/kimai/emails/)

* [Kimai - Time tracking/User feedback (Flash messages and Alerts)](https://hosted.weblate.org/projects/kimai/flashmessages/)

* [Kimai - Time tracking/Invoice-Templates](https://hosted.weblate.org/projects/kimai/invoice-templates/)

* [Kimai - Time tracking/Invoice-Calculator](https://hosted.weblate.org/projects/kimai/invoice-calculator/)

* [Kimai - Time tracking/Validations](https://hosted.weblate.org/projects/kimai/validations/)

* [Kimai - Time tracking/Actions (Dropdowns and Toolbars)](https://hosted.weblate.org/projects/kimai/actions/)

* [Kimai - Time tracking/Date-Picker](https://hosted.weblate.org/projects/kimai/daterangepicker/)

* [Kimai - Time tracking/System configurations](https://hosted.weblate.org/projects/kimai/system-configuration/)

* [Kimai - Time tracking/Wizard](https://hosted.weblate.org/projects/kimai/wizard/)

* [Kimai - Time tracking/Tasks](https://hosted.weblate.org/projects/kimai/tasks/)

* [Kimai - Time tracking/Export](https://hosted.weblate.org/projects/kimai/export/)

* [Kimai - Time tracking/Reporting](https://hosted.weblate.org/projects/kimai/reporting/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/kimai/messages/horizontal-auto.svg)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/kimai/kimai?pullRequest=5012) <br/>All committers have signed the CLA.

weblate committed:
Translated using Weblate (Catalan)

Currently translated at 2.0% (9 of 437 strings)

Translated using Weblate (Slovenian)

Currently translated at 30.7% (4 of 13 strings)

Translated using Weblate (Slovenian)

Currently translated at 97.3% (37 of 38 strings)

Translated using Weblate (Slovenian)

Currently translated at 31.5% (138 of 437 strings)

Translated using Weblate (French)

Currently translated at 99.7% (436 of 437 strings)

Translated using Weblate (German)

Currently translated at 100.0% (437 of 437 strings)

Translated using Weblate (Russian)

Currently translated at 91.9% (402 of 437 strings)

Translated using Weblate (Arabic)

Currently translated at 100.0% (4 of 4 strings)

Translated using Weblate (Arabic)

Currently translated at 83.8% (26 of 31 strings)

Translated using Weblate (Arabic)

Currently translated at 93.5% (409 of 437 strings)

Translated using Weblate (Catalan)

Currently translated at 0.0% (0 of 437 strings)

Translated using Weblate (Ukrainian)

Currently translated at 98.6% (431 of 437 strings)

Translated using Weblate (Indonesian)

Currently translated at 75.0% (3 of 4 strings)

Translated using Weblate (Indonesian)

Currently translated at 9.6% (3 of 31 strings)

Translated using Weblate (Indonesian)

Currently translated at 37.5% (3 of 8 strings)

Translated using Weblate (Indonesian)

Currently translated at 75.0% (3 of 4 strings)

Translated using Weblate (Indonesian)

Currently translated at 23.0% (3 of 13 strings)

Translated using Weblate (Indonesian)

Currently translated at 7.8% (3 of 38 strings)

Translated using Weblate (Indonesian)

Currently translated at 33.3% (3 of 9 strings)

Translated using Weblate (Indonesian)

Currently translated at 30.0% (3 of 10 strings)

Translated using Weblate (Indonesian)

Currently translated at 11.1% (3 of 27 strings)

Translated using Weblate (Indonesian)

Currently translated at 3.8% (3 of 78 strings)

Translated using Weblate (Russian)

Currently translated at 88.7% (388 of 437 strings)

Translated using Weblate (Swedish)

Currently translated at 100.0% (13 of 13 strings)

Translated using Weblate (Swedish)

Currently translated at 100.0% (38 of 38 strings)

Translated using Weblate (Swedish)

Currently translated at 100.0% (437 of 437 strings)

Co-authored-by: BeckeBauer <berko@gmx.de>
Co-authored-by: Biscuittttt <biscuitwithtea.tall310@passinbox.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Héctor Borrás Aleixandre <hborrasaleixandre@gmail.com>
Co-authored-by: John Titor <utkin2007@gmail.com>
Co-authored-by: Léane GRASSER <leane.grasser@proton.me>
Co-authored-by: Prefill add-on <noreply-addon-prefill@weblate.org>
Co-authored-by: Reza Almanda <rezaalmanda27@gmail.com>
Co-authored-by: Uporabnik Alen <obutimacek556@gmail.com>
Co-authored-by: XblateX <blate@users.noreply.hosted.weblate.org>
Co-authored-by: tygyh <jonis9898@hotmail.com>
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/ar/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/id/
Translate-URL: https://hosted.weblate.org/projects/kimai/daterangepicker/id/
Translate-URL: https://hosted.weblate.org/projects/kimai/export/id/
Translate-URL: https://hosted.weblate.org/projects/kimai/flashmessages/id/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-calculator/sl/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-calculator/sv/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/ar/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/ca/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/de/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/fr/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/ru/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/sl/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/sv/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/uk/
Translate-URL: https://hosted.weblate.org/projects/kimai/plugins/ar/
Translate-URL: https://hosted.weblate.org/projects/kimai/plugins/id/
Translate-URL: https://hosted.weblate.org/projects/kimai/system-configuration/id/
Translate-URL: https://hosted.weblate.org/projects/kimai/tasks/id/
Translate-URL: https://hosted.weblate.org/projects/kimai/teams/id/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/id/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/sl/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/sv/
Translate-URL: https://hosted.weblate.org/projects/kimai/wizard/id/
Translation: Kimai - Time tracking/Actions (Dropdowns and Toolbars)
Translation: Kimai - Time tracking/Date-Picker
Translation: Kimai - Time tracking/Export
Translation: Kimai - Time tracking/Invoice-Calculator
Translation: Kimai - Time tracking/Main application
Translation: Kimai - Time tracking/Plugins
Translation: Kimai - Time tracking/System configurations
Translation: Kimai - Time tracking/Tasks
Translation: Kimai - Time tracking/Teams
Translation: Kimai - Time tracking/User feedback (Flash messages and Alerts)
Translation: Kimai - Time tracking/Validations
Translation: Kimai - Time tracking/Wizard
